### ISSUE TITLE
Block song or artist

### ISSUE THREAD

**Issue by ProtoTipo88** [fast4x/RiMusic#1929]
### Description

I no longer want to listen to some songs or artists that are automatically suggested

### Suggested Solution

add a button 

### Alternatives

add a button in a menu

**Comment by dt192** [comment]
Yes please. This could also be implimented by long pressing the heart icon at which point it changes to a red skull or something indicating it is blocked.
